internal/component/remote/s3/s3.go

```diff
@@ -106,11 +100,11 @@ func (s *Component) Run(ctx context.Context) error {
 func (s *Component) Update(args component.Arguments) error {
 	newArgs := args.(Arguments)
 
-	s3cfg, err := generateS3Config(newArgs)
+	awsCfg, err := aws_common_config.GenerateAWSConfig(newArgs.Options.Client)
 	if err != nil {
 		return nil
```

Comment:
@rfratto pinging 'cause you authored this in the past. Do you know why this returns `nil` in the present of non-nil error? 👀

Reply:
I think remote.s3 is originally from @mattdurham; do you remember if this is intentional? It does look a little like a typo to silently ignore the error here.

Reply:
Ooof, I cant think of a good reason why I did that.